contrib/test/integration/e2e.yml

```diff
@@ -1,5 +1,8 @@
 ---
 
+- name: Set SELinux label for cri-o binary
+  command: chcon -t container_runtime_t /usr/local/bin/crio
+
```

Comment:
This is fine for testing purposes, but really I think this should be set at cri-o build/install time (from the makefile and/or RPM?).  Once that's fixed, I have NP junking 982, but in all cases the above is probably not a change we want to persist.

Reply:
(as discussed on IRC)

Reply:
:+1: The purpose here is to debug the SELinux issues.